src/main.cpp

```diff
@@ -127,13 +146,18 @@ int main(int argc, char** argv, char** envp) {
         return 1;
     }
 
+    if (GAMMA < 0 || GAMMA > 1) {
+        Debug::log(NONE, "✖ Gamma invalid: {}%. The gamma has to be between 0% and 100%", GAMMA*100);
+        return 1;
+    }
+
     if (!identity)
         Debug::log(NONE, "┣ Setting the temperature to {}K{}\n┃", KELVIN, kelvinSet ? "" : " (default)");
     else
         Debug::log(NONE, "┣ Resetting the matrix (--identity passed)\n┃", KELVIN, kelvinSet ? "" : " (default)");
 
     // calculate the matrix
-    state.ctm = identity ? Mat3x3::identity() : matrixForKelvin(KELVIN);
+    state.ctm = identity ? Mat3x3::identity() : matrixForKelvin(KELVIN, GAMMA);
```

Comment:
what if someone wants to just change gamma? We should allow that, you can just `Mat3x3::identity() * GAMMA` iirc? or maybe `.multiply`, forgot

Reply:
While fixing that, I realized there was a cleaner way to handle this, so I rewrote it and moved to #16.